python/tvm/target/target.py

```diff
@@ -494,3 +498,32 @@ def _load_config_dict(config_dict_str):
         if not isinstance(key, str):
             return None
     return config
+
+
+def refresh_host(target, host=None, target_is_key=True):
```

Comment:
let's also move this function as a static function of the Target class and document clearly that it is for legacy target pair. Let's name it consistent to that on the C++ side.